test/spec/faros_event_spec.sh

```diff
@@ -37,6 +37,8 @@ Describe 'faros_event.sh'
           --deploy_source "<deploy_source>" \
           --deploy_url "<deploy_url>" \
           --deploy_app_platform "<deploy_app_platform>" \
+          --deploy_app_tags "<key1>:<value1>,<key2>:<value2>" \
```

Comment:
What's the default value for tags and paths if nothing is passed in ? Null?

Reply:
The keys are not added to the event body.